daemon/test/client_test.go

```diff
@@ -17,6 +23,52 @@ func (suite *ClientTestSuite) SetupTest() {
 	earningClient.SendData()
 }
 
+func (suite *ClientTestSuite) TestUpsert() {
+	fmt.Println("In TestUpsert")
+	//config.InitConfig()
+	//serviceConfig := config.GetConfig()
+	dbConfig := fmt.Sprintf("host=%s user=%s dbname=%s sslmode=disable password=%s",
+		"localhost", "postgres", "postgres", "docker")
```

Comment:
@Michae-zHOU There are few errors there while I use config, so I hard code the DB connection here.

Reply:
It is ok for now. We can adjust testing later.

Reply:
Cool